CIP-????/README.md

```diff
@@ -0,0 +1,471 @@
+---
+CIP: ????
+Title: Sums-of-products in Plutus Core
+Authors: Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Status: Proposed
```

Comment:
maybe not required... only if & when there is discussion of the proposal here: I would ask that we add a reference to this PR URL in the header preamble (under `Discussions`);